job-dsls/jobs/errai_pr.groovy

```diff
@@ -31,7 +31,7 @@ job(jobName) {
     description("Runs CI build against PRs submitted to the Errai repository (github.com/errai/errai.git)")
 
     logRotator {
-        daysToKeep(7)
+        daysToKeep(3)
```

Comment:
maybe better to have last builds not the days as it is rare to have PR here

Reply:
@mareknovotny done